### ISSUE TITLE
Audio Data Annotation 

### ISSUE THREAD

**Issue by slee-ie** [sung-park/sensor_data_labeler#30]
(3) 음성데이터 추가
현재로서는 음성데이터에 대한 라벨링이 (짖기/그외 소리내기) 이루어지지 않고 있는데요,
자유실험에서는 영상에 음성이 붙지 않으므로, audio 데이터를 추가로 불러와야 할 필요가 있습니다.
audio 데이터를 붙이는 동시에 싱크도 맞출 수 있도록 해주시면 감사드리겠습니다.
뿐만아니라 audio 데이터에 대한 signal 패널도 추가되면 라벨링에 용이하게 사용될 수 있습니다.

**Comment by sung-park** [comment]
> audio 데이터에 대한 signal 패널도 추가되면 라벨링에 용이하게 사용될 수 있습니다.
네 일전에 한번 간단히 살펴본 적이 있는데 생각보다 쉽게 되지 않아서 미루고 있었습니다. 다시 한번 검토해보겠습니다.
